relay/kusama/src/governance/fellowship.rs

```diff
@@ -362,6 +362,7 @@ impl pallet_ranked_collective::Config<FellowshipCollectiveInstance> for Runtime
 	type MinRankOfClass = sp_runtime::traits::Identity;
 	type MemberSwappedHandler = ();
 	type VoteWeight = pallet_ranked_collective::Geometric;
+	type MaxMemberCount = ();
```

Comment:
ehh is this right?

Reply:
Yes i think so, as it returns `None` by default: https://github.com/paritytech/polkadot-sdk/pull/4807#discussion_r1642642279  
But i will add a test just to be sure.